.github/workflows/CI.yml

```diff
@@ -247,33 +247,23 @@ jobs:
 
     env:
       TEST_SUITES: testinstall
+      # CHERE_INVOKING=1 lets us start a 'login shell' (to set paths) without changing directory
+      CHERE_INVOKING: 1
 
-    steps:
-      # This sets git to use UNIX line endings. While GAP should allow Windows
-      # line endings in GAP files, we currently can't build GAP if the whole
-      # source has Windows line endings
-      - name: "Set git to use UNIX-style line endings"
-        run: |
-          git config --global core.autocrlf false
-          git config --global core.eol lf
 
+    steps:
       - uses: actions/checkout@v2
 
       - uses: gap-actions/setup-cygwin-for-gap@v1
 
       # CHERE_INVOKING=1 lets us start a 'login shell' (to set paths) without changing directory
       - name: "Compile GAP and download packages"
-        shell: cmd
-        run: |
-          @ECHO ON
-          SET CHERE_INVOKING=1
-          C:\cygwin64\bin\bash -l -c "bash dev/ci-prepare.sh"
+        shell: C:\cygwin64\bin\bash.exe --login --norc -o igncr '{0}'
```

Comment:
Perhaps you could explain why each of these arguments is needed? That may be quite helpful for future maintainers of this build step and the next one (e.g. you in 12 months :-))